README.md

```diff
@@ -192,10 +192,63 @@ options.automation_name = 'safari'
 # calls to it could be chained
 options.set_capability('browser_name', 'safari')
 
-# Appium1 points to http://127.0.0.1:4723/wd/hub by default 
+# Appium1 points to http://127.0.0.1:4723/wd/hub by default
 self.driver = webdriver.Remote('http://127.0.0.1:4723', options=options, strict_ssl=False)
 ```
 
+## Set custom `AppiumConnection`
+
+The first argument of `webdriver.Remote` can set an arbitrary command executor for you.
+
+1. Set init arguments for the pool manager Appium Python client uses to manage http requests.
+
+```python
+from appium import webdriver
+from appium.options.ios import XCUITestOptions
+
+import urllib3
+from appium.webdriver.appium_connection import AppiumConnection
+
+# Retry connection error up to 3 times.
+init_args_for_pool_manage = {
+    'retries': urllib3.util.retry.Retry(total=3, connect=3, read=False)
+}
+appium_executor = AppiumConnection(remote_server_addr='http://127.0.0.1:4723',
+    init_args_for_pool_manage=init_args_for_pool_manage)
+
+options = XCUITestOptions().load_capabilities({
```

Comment:
load_capabilities was created rather for compatibility with older stuff. Consider using appropriate setters and getters instead